languages/java/config.json

```
@@ -57,6 +57,12 @@
         "uuid": "2165c331-6b0c-49b8-9555-bad577ec4fc6",
         "concepts": ["strings"],
         "prerequisites": ["basics"]
+      },
+      {
+        "slug": "conditionals-ternary",
+        "uuid": "2165c331-6b0c-49b8-9555-bad577ec4fc6",
+        "concepts": ["strings"],
+        "prerequisites": ["conditionals", "numbers"]
```

The `design.md` document lists `conditionals-if` as the prerequiste (which has the same naming problem I mentioned above by the way), so I think this should be:

```suggestion
        "prerequisites": ["conditionals-if", "numbers"]
```